stubs/ORM/Query.stub

```diff
@@ -3,10 +3,24 @@
 namespace Doctrine\ORM;
 
 /**
+ * @template TKey as array-key
```

Comment:
This might be conflicting with the solution propsed in https://github.com/phpstan/phpstan-doctrine/pull/384